environment.yml

```diff
@@ -17,4 +17,4 @@ dependencies:
 - wget
 - pip:
   - rooki
-  - multiurl
+  - multiurl>=0.2.3.2
```

Comment:
multiurl is on conda-forge, and `0.2.3.2` is already available so I would install using conda rather than pip